com.CalculaPro.app/calcula-pro.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.8.4
```

Comment:
the `version` field has nothing to do with the application version. This value is invalid.